folly/io/async/AsyncServerSocket.cpp

```diff
@@ -736,7 +736,11 @@ int AsyncServerSocket::createSocket(int family) {
   return fd;
 }
 
+#ifndef TCP_NOPUSH
 void AsyncServerSocket::setupSocket(int fd, int family) {
+#else
+void AsyncServerSocket::setupSocket(int fd, int) {
+#endif
   // Put the socket in non-blocking mode
```

Comment:
The less conditional compilation, the better.

Instead, just stick this as the first line:

```
(void)family;
```

Reply:
Ah good point. Just saw this after my similar change this morning. I'll just stick it as the first line for those two functions who have conditional unused parameters. I'll send a change up tonight.